components/post_view/reaction_list/reactions_list.test.jsx

```diff
@@ -2,9 +2,11 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import {shallow} from 'enzyme';
+import { shallow } from 'enzyme';
 
 import ReactionList from './reaction_list.jsx';
+import { EmojiIndicesByAlias } from '../../../utils/emoji';
```

Comment:
Can you change this be absolute import path? anymore than `./` we usually prefer absolute imports